### PR TITLE
[stackwalker] add new port

### PR DESCRIPTION
**Pull request by AenBleidd**
If this PR adds a new port, please uncomment and fill out this checklist:

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines.
- [x] Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html)
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [x] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/stackwalker/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO JochenKalmbach/StackWalker
+    REF "${VERSION}"
+    SHA512 ce4004f114400ff66e25d7403c52ed4798a0e94d529335995df525f37d4238c750c8b1ee5801f71bd7128d39baa9af18e546a49da8587976720df6e9b372b851
+    HEAD_REF master
+    PATCHES
+        fix-build.patch
+        fix-exports.patch
```
Comment (FrankXie05) [inline ports/stackwalker/portfile.cmake:11]: Need to correctly link to the location of the header file for the usage of cmake.
```
E:\NEW-PORT\main.cpp(2,10): fatal error C1083: Cannot open include file: 'StackWalker.h': No such file or directory [E: \NEW-PORT\build\main.vcxproj]
```
https://github.com/JochenKalmbach/StackWalker/blob/5b0df7a4db8896f6b6dc45d36e383c52577e3c6b/CMakeLists.txt#L81

Main.cpp:
```
#include <windows.h>
#include "StackWalker.h"

void Func5() { StackWalker sw; sw.ShowCallstack(); }
void Func4() { Func5(); }
void Func3() { Func4(); }
void Func2() { Func3(); }
void Func1() { Func2(); }

int main()
{
    Func1();
    return 0;
}
```

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: ports/stackwalker/portfile.cmake (hunk 1 above)
Comment (AenBleidd) [reply ports/stackwalker/portfile.cmake:11]: Should be fine now

**Comment by AenBleidd** [comment]
@FrankXie05, please review

**Comment by FrankXie05** [comment]
The usage has been tested successfully locally.

**Review by FrankXie05 — DISMISSED** [review]

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the new port! Here's hoping upstream drops a release soon!

**Comment by AenBleidd** [comment]
> Here's hoping upstream drops a release soon!

Me as well :-D

**Pull request merged**